guess_game.py

```diff
@@ -0,0 +1,46 @@
+import random
+
+
+def guessGame(num):
+    print(f'Guess a number between 1 and {num}.You have got 3 attempts to guess!!')
+    user_guess = int(input('Enter the number :'))
+    comp_choose = random.randint(1, num)
+    attempts = 1
+    while user_guess != comp_choose and attempts < 3:
+        if user_guess < 1 or user_guess > num:
+
+            try:
```

Comment:
Remobe try except block and convert to integer just after the while loop instead.